packages/fxa-content-server/grunttasks/clean.js

```diff
@@ -11,8 +11,6 @@ module.exports = function (grunt) {
           dot: true,
           src: [
             '<%= yeoman.tmp %>/*',
-            // css files are regenerated on every server run.
-            '<%= yeoman.app %>/styles/**/*.css',
```

Comment:
I removed this because otherwise, `tailwind.css` was getting removed from `fxa-content-server/app/styles` every time I tried to `yarn build`, and when I compared files (in `app` as well as `dist`) before and after removing this, I actually don't see what it's doing for us anyway since the only other generated CSS file is `main.css` 🤷‍♀️